### PR TITLE
Add wisdom.js for dmz250 dimmer

### PR DESCRIPTION
**Pull request by SjonHortensius**

### CODE REVIEW

File: devices/wisdom.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const tuya = require('zigbee-herdsman-converters/lib/tuya');
+const utils = require('zigbee-herdsman-converters/lib/utils');
+const globalStore = require('zigbee-herdsman-converters/lib/store');
+const e = exposes.presets;
+const ea = exposes.access;
+
+const definition = {
+    zigbeeModel: ['HK-DIM-SW'],
+    model: 'dmz250',
+    vendor: 'Wisdom',
```
Comment (Koenkk) [inline devices/wisdom.js:16]: Looks this device is from [Profulux](https://profolux.com/product/led-dimmer-zigbee-3-0-5-250w/). Can you move it to `profulux.js`?

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1769

File: devices/wisdom.js (hunk 1 above)
Comment (SjonHortensius) [reply devices/wisdom.js:16]: @Koenkk is fine by me, the device identifies itself as Wisdom - so I actually think Profolux is simply a reseller?

Comment (SjonHortensius) [reply devices/wisdom.js:16]: also, did you notice there is no profOlux yet (not profUlux), there is only profAlux which is yet another manufacturer

Comment (Koenkk) [reply devices/wisdom.js:16]: Can you provide a link of where you bought the device?

Comment (SjonHortensius) [reply devices/wisdom.js:16]: that was through https://www.ledlampaanbiedingen.nl/zigbee-smart-wifi-led-dimmer-0-250-watt-werkt-via.html but it seems they are both white-labeling the same hardware. Again, the device identifies itself as `Wisdom` which is why I picked it

Comment (Koenkk) [reply devices/wisdom.js:16]: I see, then let's keep it as is.

**Pull request merged**